services/dy-3dvis/docker-compose.yml

```diff
@@ -1,28 +1,94 @@
-version: '3.4'
+version: '2.3'
```

Comment:
Any reason for this? Do we actually have some kind of rules here?

Reply:
yes, the __runtime__ field is not available in version 3 of compose... necessary to enable the nvidia runtime.